lib/routes.js

```diff
@@ -113,6 +113,12 @@ publicRoutes.route('/posts/:postId', {
   }
 });
 
+publicRoutes.route('/kontakt', {
+ action: () => {
+   BlazeLayout.render('kontakt');
```

Comment:
Are you sure you want to render it directly without using any layout? like the blogLayout (which adds the blog header) @dagvonkr ?